tests/unit/controllers/v0.3/DependentAccountAPI.test.js

```diff
@@ -234,12 +253,26 @@ describe("DependentAccountAPI", () => {
     });
   });
 
+  describe("checkAccountExpiration", () => {
```

Comment:
Some tests aren't new but I just moved them around so it appears as if they are new. This is just to match up the order of the function declaration in the class' file. It just makes it easier to read for me when the function and tests for that function are in the same order.